### PR TITLE
Update css_basics demo

### PR DESCRIPTION
**Pull request by cedlemo**

### CODE REVIEW

File: gtk3/sample/gtk-demo/css_basics.rb [hunk 1]
```diff
@@ -1,103 +1,76 @@
...
-        css_text_changed(_text, provider)
-      end
-
-      text.text = File.read(File.join(__dir__, "css_basics.css"))
-
-      provider.signal_connect("parsing-error") do |_provider, section, error|
-        p section
-        p error
-        show_parsing_error(section, error, child.buffer)
+def apply_style(widget, provider)
```
Comment (kou) [inline gtk3/sample/gtk-demo/css_basics.rb]: This method should be defined in `CssBasicDemo` because this method is only used in `CssBasicDemo`.

Comment (cedlemo) [inline gtk3/sample/gtk-demo/css_basics.rb]: Ok,
so I guess I have to do the same for the css_accordion.rb demo I have done previously :-)
https://github.com/ruby-gnome2/ruby-gnome2/blob/master/gtk3/sample/gtk-demo/css_accordion.rb

Comment (kou) [inline gtk3/sample/gtk-demo/css_basics.rb]: > so I guess I have to do the same for the css_accordion.rb demo I have done previously :-)

Yes, please!

**Comment by cedlemo** [comment]
Done

**Comment by kou** [comment]
Thanks!!!!

**Pull request merged**